tide_news.install

```diff
@@ -5,801 +5,13 @@
-    $entity_view_display->save();
-  }
-
-  $entity_view_display = \Drupal::entityTypeManager()
-    ->getStorage('entity_view_display')
-    ->load('node.news.featured_card');
-  if ($entity_view_display) {
-    $components = $entity_form_display->getComponents();
-    $hidden = $entity_view_display->get('hidden');
-    $hidden['field_landing_page_nav_title'] = TRUE;
-    $hidden['field_show_site_section_nav'] = TRUE;
-    $hidden['field_show_topic_term_and_tags'] = TRUE;
-    $entity_view_display->set('hidden', $hidden);
-    $entity_view_display->save();
-  }
-
-  $config_factory = \Drupal::configFactory();
-  $config = $config_factory->getEditable('jsonapi_extras.jsonapi_resource_config.node--news');
-
-  $content = $config->get('resourceFields');
-  if (!isset($content['field_landing_page_nav_title'])) {
-    $content['field_landing_page_nav_title'] = [
-      'fieldName' => 'field_landing_page_nav_title',
-      'publicName' => 'field_landing_page_nav_title',
-      'enhancer' => [
-        'id' => '',
-      ],
-      'disabled' => FALSE,
-      'advancedOptionsIcon' => '',
-      'enhancer_label' => '',
-    ];
-
-    $config->set('resourceFields', $content);
-  }
-  if (!isset($content['field_show_site_section_nav'])) {
-    $content['field_show_site_section_nav'] = [
-      'fieldName' => 'field_show_site_section_nav',
-      'publicName' => 'field_show_site_section_nav',
-      'enhancer' => [
-        'id' => '',
-      ],
-      'disabled' => FALSE,
-      'advancedOptionsIcon' => '',
-      'enhancer_label' => '',
-    ];
-
-    $config->set('resourceFields', $content);
-  }
-  if (!isset($content['field_show_topic_term_and_tags'])) {
-    $content['field_show_topic_term_and_tags'] = [
-      'fieldName' => 'field_show_topic_term_and_tags',
-      'publicName' => 'field_show_topic_term_and_tags',
-      'enhancer' => [
-        'id' => '',
-      ],
-      'disabled' => FALSE,
-      'advancedOptionsIcon' => '',
-      'enhancer_label' => '',
-    ];
-
-    $config->set('resourceFields', $content);
-  }
-
-  $config->save();
-}
-
-/**
- * Use Select2 for Location and Department field.
- */
-function tide_news_update_8008() {
-  if (\Drupal::moduleHandler()->moduleExists('select2')) {
-    $config_factory = \Drupal::configFactory();
-    $config = $config_factory->getEditable('core.entity_form_display.node.news.default');
-    $content = $config->get('content');
-    foreach (['field_location', 'field_node_department'] as $select2_field) {
-      if (isset($content[$select2_field])) {
-        $content[$select2_field]['type'] = 'select2_entity_reference';
-        $content[$select2_field]['settings'] = [
-          'match_operator' => 'CONTAINS',
-          'match_limit' => 10,
-          'autocomplete' => FALSE,
-          'width' => '100%',
-        ];
-      }
-    }
-    $config->set('content', $content);
-    $config->save();
-  }
+  TideNewsOperation::addToWorkflows();
+  TideNewsOperation::addToShceduledTransitions();
```

Comment:
Still a typo here :(

`addToShceduledTransitions` should be `addToScheduledTransitions`

Reply:
my god, I am going blind. Good one. Let me fix it for one more time :D